Source/WebCore/platform/Length.h

```diff
@@ -231,6 +231,11 @@ inline Length& Length::operator=(Length&& other)
     return *this;
 }
 
+inline bool operator<(const Length& lhs, const Length& rhs)
```

Comment:
Why is this operator needed?

Reply:
It is not needed for this patch anymore. While I was working on a solution I added this since it was needed, but eventually I no longer required it. It seemed like it would be a useful operator to have since it would allow us to use std::min and std::max on Length objects. However, if you think we should remove it and only add it when it is required I can make the adjustment.

Reply:
Yes, it is better to not include unused code in patches. Also one less thing to review! :)

Reply:
Sounds good, removed!